tests/trestle/core/profile_resolver_test.py

```diff
@@ -244,6 +252,39 @@ def test_parameter_resolution(tmp_trestle_dir: pathlib.Path) -> None:
     assert len(locations_a) == 1
 
 
+def test_merge_params() -> None:
+    """Test the merge of params."""
+    params: List[com.Parameter] = test_utils.generate_param_list('foo', 2)
+    profile = gens.generate_sample_model(prof.Profile)
+    method = prof.Method.merge
+    combine = prof.Combine(method=method)
+    profile.merge = prof.Merge(combine=combine)
+
+    merge = ProfileResolver.Merge(profile)
+    merge._merge_params(params[0], params[1])
+    # FIXME this should check the right things happened
```

Comment:
should this be fixed? Please add name or issue for the future reference or it will likely be lost in time

Reply:
I removed that comment because previously nothing was being checked, but now there are checks on counts that confirm the merge happened.